modules/installation/pages/install-uyuni.adoc

```diff
@@ -0,0 +1,79 @@
+[[install-uyuni]]
+= Install Uyuni Server with openSUSE
+
+{productname} Server can be installed on openSUSE.
+
+[NOTE]
+====
+This table specifies the minimum requirements.
+====
+
+In this table, replace ``version`` with the version of the product you are using.
```

Comment:
What does this mean? I don't see ```version``` at the table.

Reply:
I commented it.